network-api/networkapi/buyersguide/templates/bg_base.html

```diff
@@ -1,23 +1,25 @@
 {% extends "pages/base.html" %}
 
-{% load bg_nav_tags %}
+{% load bg_nav_tags localization i18n %}
 
+{% get_current_language as lang_code %}
 
-{% block page_title %}{% if pageTitle %}{{ pageTitle }}{% else %}Mozilla - *privacy not included{% endif %}{% endblock %}
+{% block page_title %}{% if pageTitle %}{{ pageTitle }}{% else %}Mozilla - {% blocktrans context "This can be localized. This is a reference to the “*batteries not included” mention on toys." %}*privacy not included{% endblocktrans %}{% endif %}{% endblock %}
 
 
 {# TODO: consider using a different ga_identifier? #}
 
 
 {% block social_metadata %}
   <meta property='og:type' content='website'>
-  <meta property='og:locale' content='en-US'>
-  <meta property="og:title" content="*Privacy Not Included: A Buyer’s Guide for Connected Products">
+  <meta property='og:locale' content='{{ lang_code|to_opengraph_locale }}'>
+  <meta property="og:title" content='{% trans "*Privacy Not Included: A Buyer’s Guide for Connected Products" context "*Privacy Not Included can be localized. This is a reference to the “*batteries not included” mention on toys." %}'>
+  <meta property='wagtail:language' content='{{ lang_code }}'>
 {% endblock %}
 
 
 {% block wagtail_metadata %}
-  <meta property="og:description" content="Smart home gadgets, fitness trackers, toys and more, rated for their privacy & security">
+  <meta property="og:description" content='{% trans "Smart home gadgets, fitness trackers, toys and more, rated for their privacy & security" %}'>
```

Comment:
this should be able to stay `content="{% .... %}"`, the HTML will never see the quotes in the templating instructions

Reply:
@TheoChevalier can this one still be fixed?

Reply:
Thanks, missed it for some reason. Fixed!